json-schemas/src/app-stats.json

```
@@ -9,632 +9,786 @@
     },
     "unit": {
       "type": "string",
-      "description": "unit of time for these stats from the intervalId forwards",
+      "description": "Unit of time for these stats from the intervalId forwards.",
       "pattern": "^minute$"
     },
     "schema": {
       "type": "string",
-      "description": "URI to this schema"
+      "description": "URI to this schema."
     },
     "entries": {
       "type": "object",
-      "description": "stats entries",
+      "description": "Stats entries.",
       "properties": {
         "messages.all.all.count": {
           "type": "number",
-          "inclusiveMinimum": 0
+          "inclusiveMinimum": 0,
+          "description": "Total message count."
         },
         "messages.all.all.data": {
           "type": "number",
-          "inclusiveMinimum": 0
+          "inclusiveMinimum": 0,
+          "description": "Total message size."
         },
         "messages.all.all.uncompressedData": {
           "type": "number",
-          "inclusiveMinimum": 0
+          "inclusiveMinimum": 0,
+          "description": "Total uncompressed message size (excluding any compression, e.g., delta compression: https://ably.com/documentation/realtime/channels/channel-parameters/deltas)."
         },
         "messages.all.messages.count": {
           "type": "number",
-          "inclusiveMinimum": 0
+          "inclusiveMinimum": 0,
+          "description": "Total message count, excluding presence messages."
         },
         "messages.all.messages.data": {
           "type": "number",
-          "inclusiveMinimum": 0
+          "inclusiveMinimum": 0,
+          "description": "Total message size, excluding presence messages."
         },
         "messages.all.messages.uncompressedData": {
           "type": "number",
-          "inclusiveMinimum": 0
+          "inclusiveMinimum": 0,
+          "description": "Total uncompressed message size (excluding any compression, e.g., delta compression: https://ably.com/documentation/realtime/channels/channel-parameters/deltas), excluding presence messages."
```

Here, and elsewhere, you don't need the comma after "e.g.". i.e.:

```suggestion
          "description": "Total uncompressed message size (excluding any compression, e.g. delta compression: https://ably.com/documentation/realtime/channels/channel-parameters/deltas), excluding presence messages."
```